src/structures/Interaction.js

```diff
@@ -269,10 +269,70 @@ class Interaction extends Base {
    * Indicates whether this interaction is a {@link SelectMenuInteraction}.
    * @returns {boolean}
    */
+  isAnySelectMenu() {
+    return InteractionTypes[this.type] === InteractionTypes.MESSAGE_COMPONENT && typeof this.values !== 'undefined';
+  }
+
+  /**
+   * Indicates whether this interaction is a {@link SelectMenuInteraction}
+   * @returns {boolean}
+   * @deprecated Use {@link Interaction#isStringSelect()} instead
+   */
   isSelectMenu() {
+    return this.type === InteractionTypes.STRING_SELECT;
```

Comment:
This is completely wrong now... `this.type` is the InteractionTypes value and refers to `MESSAGE_COMPONENT` for a `StringSelectMenuInteraction`, so this will always return false now. It should do the same as `isStringSelect()` does and could even simply call that function internally to not have code duplication.

Reply:
Yup,sorry for my mistake
Resoved by: https://github.com/discordjs/discord.js/pull/8814/commits/4123b7640566871f5396071d301a9989e440199c